src/EventListener/SimplifiedApiProblemListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ApiProblemBundle\Event\CreateApiProblem;
+use Throwable;
+
+trait SimplifiedApiProblemListener
```

Comment:
These are problem better upstream.

Reply:
Can be extracted once confirmed and stable